Description: Fix for change in SWIG after 1.3.36
 Upstream 0.2x branch is essentially unmaintained.
Forwarded: no
Author: Mark Ellis <mark@mpellis.org.uk>
Last-Update: 2010-05-24

diff -Nurp opensync-0.22.orig/wrapper/opensync.i opensync-0.22/wrapper/opensync.i
--- opensync-0.22.orig/wrapper/opensync.i	2007-03-27 12:49:09.000000000 +0100
+++ opensync-0.22/wrapper/opensync.i	2010-05-21 18:36:46.857447188 +0100
@@ -82,7 +82,7 @@ typedef struct {} OSyncHashTable;
 %extend OSyncChange {
 	OSyncChange(PyObject *obj=NULL) {
 		OSyncChange *change = NULL;
-		if (obj)
+		if ((obj) && (obj != Py_None))
 			change = (OSyncChange *)PyCObject_AsVoidPtr(obj);
 		else
 			change = osync_change_new();
